parlai/agents/jit/README.md

```diff
@@ -0,0 +1,25 @@
+# Agent exported to TorchScript (JIT compilation)
+
+This agent will read in a ParlAI agent that has been exported to TorchScript with JIT compilation, for use in greedy-search inference on CPU. This allows inference to be run on models without using any ParlAI overhead, either for tokenization or for the forward passes through the model. Currently, only BART models are supported.
```

Comment:
are there plans to support all `transformer/generator`s? if not, should we name this `bart_jit`?

Reply:
yeah, I definitely would like to see us support more than just BART, especially once we see how well TorchScripting works for the BART models